wagtail/contrib/modeladmin/static_src/wagtailmodeladmin/scss/index.scss

```diff
@@ -4,6 +4,22 @@
     margin-bottom: 0;
 }
 
+.result-count {
+    display: block;
+    font-weight: 500;
+
+    &:before {
```

Comment:
Question: Any reason for doing these via CSS rather than just baking into the template?

Reply:
Yes. To give people more options for changing the appearance in the future. If there is one thing I dream of, it is an admin that is easier to visually customize.

Reply:
It is probably easier to override `modeladmin/includes/result_count.html` in your app's templates dir, than injecting custom CSS and IMHO it is a more Django-like approach

Reply:
I dream of a Wagtail with well-written CSS and consistent markup. Until then, I'd like to make it as easy as possible for everyone to find ways to handle it.

Reply:
> I dream of a Wagtail with well-written CSS and consistent markup.

I'd like to hear your hear your thoughts on that either in a separate issue or Slack, especially anything not covered by #3804. There is most definitely room for improvement and we are chipping away at it slowly. Any and all improvement PRs are welcome too ;)

Reply:
They'll come — thoughts and PRs as well. This particular PR was just small enough and useful enough to rush in.